website/docs/ProtractorMigration.md

```diff
@@ -0,0 +1,108 @@
+
+## Setup
+
+The Protractor and WebdriverIO API is actually very similar, to a point where the majority of commands can be rewritten in an automted way through a [codemod](https://github.com/webdriverio/codemod). For this tutorial we use a Protractor [boilerplate project](https://github.com/webdriverio/protractor-jasmine-typescript) and fully migrate it to WebdriverIO.
+
+To install the codemod, run:
+
+```sh
+$ npm install jscodeshift @wdio/codemod
+```
+
+#### Commits:
+
+- _install codemod_ [[0fdc214]](https://github.com/webdriverio/protractor-jasmine-typescript/commit/0fdc2146bd74e5ff9e8478c2836ae9454b0e2fb1)
+
+## Strategy
+
+There are many migration strategies. Depending on the size of your team, amount of test files and the urgency to migrate you can try to transform all tests at once or file by file. Given that Protractor will continued to be maintained until Angular version 15 (end of 2022) you still have enough time. You can have Protractor and WebdriverIO tests running at the same time and start writing new tests in WebdriverIO. Given your time budget you can then start migrating the important test cases first and work your way down to tests you might even can delete.
+
+## First the Config File
+
+After we have installed the codemod we can start transforming the first file. Have a look first into [WebdriverIOs configuration options](./Options.md). Config files can become very complex and it might make sense to only port the essential parts and see how the rest can be added once the corresponding tests that need certain options are being migrated.
+
+For the first migration we only transform the config file and run:
+
+```sh
+$ npx jscodeshift -t ./node_modules/@wdio/codemod/protractor ./conf.ts
+```
+
+:::info
+
+This migration tutorial uses an example Protractor [boilerplate project](https://github.com/webdriverio/protractor-jasmine-typescript). Your config can be named differently, however the principle should be the same: start migration the config first.
+
+:::
+
+#### Commits:
+
+- _transform config_ [[1107f9d]](https://github.com/webdriverio/protractor-jasmine-typescript/commit/1107f9d1e0a94c3de4c78e7fa1c43e5129500b52)
+
+## Install WebdriverIO Dependencies
+
+Next step is to configure a minimal WebdriverIO setup that we start building up as we migrate from one framework to another. First we install the WebdriverIO CLI via:
+
+```sh
+$ npm install --save-dev @wdio/cli
+```
+
+Next we run the configuration wizard:
+
+```sh
+$ npx wdio config
+```
+
+This will walk you through a couple of questions. For this migration scenario you pick the default choices, we recommend not to auto-generate example files, pick a different folder for WebdriverIO files and to choose Mocha above Jasmine. Even though you might have been using Protractor with Jasmine before, Mocha however provides better retry mechanisms. The choice is yours!
```

Comment:
Maybe change it to this so it reads a bit easier?

```suggestion
This will walk you through a couple of questions. For this migration scenario you:
- pick the default choices
- we recommend not to auto-generate example files
- pick a different folder for WebdriverIO files
- and to choose Mocha above Jasmine. 

:::info Why Mocha?
Even though you might have been using Protractor with Jasmine before, Mocha however provides better retry mechanisms. The choice is yours!
:::
```